editoast/src/views/path/projection.rs

```diff
@@ -206,10 +206,32 @@ impl<'a> PathProjection<'a> {
     }
 }
 
+#[cfg_attr(test, derive(Debug, PartialEq))]
+pub struct Intersection {
```

Comment:
If this is meant to be used specifically here, to represent the intersection of a track range and a path perhaps add some documentation:
-how the 2 bounds _start_ and _end_ reference a path on which we projected a track range
- these are distances relatively to the beginning of the path

Reply:
Does something like [08a956c](https://github.com/OpenRailAssociation/osrd/pull/8897/commits/08a956cf2333c3f5caee8372ba9d9a0a82081782) answer you enquiry?

I didn't document it because it is not (yet?) exposed on the API. But yet, the entire idea of this PR has been for #8794, and also, even if not exposed in the API, still good to document, especially since "from the beginning of the path" is quite important and not necessarily obvious.

Reply:
thanks, looking good